app/src/main/java/com/asfoundation/wallet/ui/onboarding/OnboardingPresenter.kt

```diff
@@ -42,7 +43,7 @@ class OnboardingPresenter(private val disposables: CompositeDisposable,
     disposables.add(referralInteractor.getReferralInfo()
         .subscribeOn(networkScheduler)
         .observeOn(viewScheduler)
-        .doOnSuccess { view.setupUi(it.symbol + it.maxAmount) }
+        .doOnSuccess { view.updateUI(it.symbol + it.amount.multiply(BigDecimal(it.available))) }
```

Comment:
My bad, it seems like it’s the total (including the invite bonus). But don’t do it just yet, as we need a new string for the case where there’s no network.